config/profile.go

```diff
@@ -25,3 +36,80 @@ func (c *Config) RemoveProfile(name string) {
 
 	delete(c.Profiles, name)
 }
+
+func (c *Config) ParseProfileInput(input string) (conn, key string, err error) {
+	c.Lock()
+	defer c.Unlock()
+
+	sepIdx := strings.LastIndex(input, profileSeparator)
```

Comment:
`LastIndex` maybe buggy here? What if `s3:/abc::/def`?

Reply:
Yes, you are right. I think it's better to use `strings.Index`, and add a strict for profile name, which cannot contain a `:` in it. 

So that we can split string before the first `:` as profile name.